weblab/experiments/tests/test_views.py

```diff
@@ -892,6 +897,93 @@ def test_view_experiment_version(self, client, experiment_version):
         assert response.context['version'] == experiment_version
 
 
+@pytest.mark.django_db
+class TestExperimentTasks:
+
+    @pytest.mark.usefixtures('other_user')
```

Comment:
I don't think we need the fixture here; the client uses an anonymous user by default unless the `logged_in_user` fixture is used.

Reply:
thanks - changed